fix code and note blocks in dualtor pages (#1649)
tigera/docs: calico-enterprise/networking/configuring/dual-tor.mdx

EarlyNetworkingConfiguration will be superseded by any BGPPeer or BGPConfiguration
resources after successful startup.

> For a full description of the EarlyNetworkingConfiguration resource, see [here](../../reference/resources/earlynetworkconfiguration.mdx)

Reviewer: ```suggestion
For a full description, see [Early Network Configuration](../../reference/resources/earlynetworkconfiguration.mdx)
```

Reviewer: What's the difference between EarlyNetworkingConfiguration (line 473) and EarlyNetworkConfiguration?

Author: EarlyNetworkingConfiguration doesn't exist - typo!
